src/mumble/GlobalShortcut_win.h

```diff
@@ -82,6 +82,16 @@ class GlobalShortcutWin : public GlobalShortcutEngine {
 		static LRESULT CALLBACK HookKeyboard(int, WPARAM, LPARAM);
 		static LRESULT CALLBACK HookMouse(int, WPARAM, LPARAM);
 
+		/// Handle an incoming Windows keyboard message.
+		///
+		/// Returns true if the button was suppressed, otherwise returns false.
```

Comment:
Hm. But unless I misread the code yesterday this function's return value only indicates whether it wants the caller to suppress what it just handled. Neither does it perform the suppression itself nor has it happened at the point the function is called. No?